src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import duke.Parser;
+import duke.Storage;
+import duke.Ui;
+import duke.command.Command;
+import duke.task.TaskList;
+
+/**
+ * This class represents Duke who runs the main program.
+ */
```

Comment:
Perhaps a short description of what Duke/the main program does could be included?